package.json

```diff
@@ -18,10 +18,12 @@
     "react": "^18.2.0",
     "react-datepicker": "^4.10.0",
     "react-dom": "^18.2.0",
+    "react-helmet": "^6.1.0",
```

Comment:
Please explain how helmet works next time.

Reply:
Removed this as we are now copying the url